PluginsAndFeatures/azure-toolkit-for-intellij/resources/META-INF/plugin.xml

```diff
@@ -20,26 +20,40 @@
   <change-notes>
     <![CDATA[
     <html>
-    <h3>[3.19.0]</h3>
-      <h4>Added</h4>
-      <ul>
-        <li>Support open browser after Web App deployment.</li>
-        <li>Support to link SQL Server Big Data cluster and submit Spark jobs.</li>
-        <li>Support WebHDFS storage type to submit job to HDInsight cluster with ADLS Gen 1 storage account.</li>
-      </ul>
-      <h4>Changed</h4>
-      <ul>
-        <li>Update UI of Web App creation and deployment</li>
-        <li>Subscription ID need to be specified for ADLS Gen 1 storage type</li>
-      </ul>
-      <h4>Fixed</h4>
-      <ul>
-        <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2840" rel="nofollow">#2840</a> Submit successfully with invalid path for WebHDFS storage type issue.</li>
-        <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2747" rel="nofollow">#2747</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2801" rel="nofollow">#2801</a> Error loadig HDInsight node issue.</li>
-        <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2714" rel="nofollow">#2714</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2688" rel="nofollow">#2688</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2669" rel="nofollow">#2669</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2728" rel="nofollow">#2728</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2807" rel="nofollow">#2807</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2808" rel="nofollow">#2808</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2811" rel="nofollow">#2811</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2831" rel="nofollow">#2831</a>Spark Run Configuration validation issues.</li>
-        <li><a href="https://github.com/Microsoft/azure-tools-for-java/issues/2810" rel="nofollow">#2810</a>,<a href="https://github.com/Microsoft/azure-tools-for-java/issues/2760" rel="nofollow">#2760</a> Spark Run Configuration issues when created from context menu.</li>
-      </ul>
-      <p>You may get the full change log <a href="https://github.com/Microsoft/azure-tools-for-java/blob/develop/CHANGELOG.md">here</a></p>
+    <h3>[3.20.0]</h3>
+    <h4>Added</h4>
+    <ul>
```

Comment:
Shall we have a little program in CI generate this html file automatically?

Reply:
Yes, may be we could generate this html from CHANGELOG.md, but we still need to write the chagelog markdown

Reply:
I mean from md to html